File: impar_par.py
import os
os.system("cls")

# dados
soma = 0
par = 0
impar = 0
# processo
for i in range (1, 6):
    numero = int(input(f"Digite o {i}º número: "))
   
    if numero % 2 == 0:
        par += 1
    else:
        impar += 1
    print(f"Números pares: {par}; Números impares: {impar}\n")

File: impar_par.js
const Numeros = [2, 10, 3, 6, 7, 8]
let par = 0
let impar = 0
let neutro = 0

for (let i = 0; i <= 5; i++) {
    if (Numeros[i] == 0) {
        neutro++
    } else if (Numeros[i] % 2 == 0) {
        par++
    } else {
        impar++
    }
}
if (neutro > 0)
    console.log(`${par} são pares, ${impar} são ímpares e ${neutro} são neutros. `)
else
    console.log(`${par} são pares e ${impar} são ímpares.`)
